gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EntityTagsController.java

```diff
@@ -52,4 +55,50 @@ public Map get(HttpServletRequest request) {
 
     return entityTagsService.get(id);
   }
+
+  @ApiOperation(value = "Deletes a subset of tags for the provided entityId",
+                notes = "multiple tags can be deleted for an entity using a comma as a separator, e.g. /tag1,tag2")
+  @RequestMapping(value = "/tags/{entityId}/{tag}", method = RequestMethod.DELETE)
+  public Map delete(@PathVariable String entityId, @PathVariable String tag) {
+    Map<String, Object> operation = new HashMap<>();
+    List<Map<String, Object>> jobs = new ArrayList<>();
+    Map<String, Object> job = new HashMap<>();
+    job.put("type", "deleteEntityTags");
+    job.put("entityId", entityId);
+    job.put("tags", tag.split(","));
+    jobs.add(job);
+    operation.put("job", jobs);
+    return taskService.create(operation);
+  }
+
+  @RequestMapping(method = RequestMethod.POST)
+  public Map post(@RequestParam("entityId") String entityId,
+                  @RequestParam(value = "entityType", required = false) String entityType,
```

Comment:
Seems like it might be less error prone to require `account`, `region` and `cloudProvider` but allow a `*` value.

`entityType` is always a required value IMO.